nav2_mppi_controller/src/critics/constraint_critic.cpp

```diff
@@ -66,12 +66,13 @@ void ConstraintCritic::score(CriticData & data)
         std::move(out_of_turning_rad_motion)) *
         data.model_dt, {1}, immediate) * weight_, power_);
   }
-
-  data.costs += xt::pow(
-    xt::sum(
-      (std::move(out_of_max_bounds_motion) +
-      std::move(out_of_min_bounds_motion)) *
-      data.model_dt, {1}, immediate) * weight_, power_);
+  else {
```

Comment:
Linting issue, if nothing else `} else {`

Reply:
i'll fix it and add pull request to main. Just wanted to double check that it's not done on purpose. I'll add a return instead of an else statement and will create a new pull request to main.